NAMESPACE

```diff
@@ -25,13 +30,13 @@ export(kozakSequenceScore)
 export(map_to_GRanges)
 export(meta_position)
 export(reassignTSSbyCage)
+export(reduce_keep_naming)
```

Comment:
This should be rather overwriting the standard reduce from the GRanges, allow for the same plus keep.names = TRUE.